tests/model_tests.cpp

```diff
@@ -164,11 +162,10 @@ TEST_CASE("test skipped fields") {
     REQUIRE(foo.barp->isOK);
     REQUIRE_THAT(foo.barp->msg, Catch::Matchers::Equals("hello world!"));
 
-    char json1[1024];
     size_t jsonlen;
-    CHECK(json_from_Foo(&foo, json1, sizeof(json1), &jsonlen) == 0);
+    char *json1 = Foo_to_json(&foo, 0, &jsonlen);
```

Comment:
None of the tests/samples I see here demonstrate how to determine whether or not the encoding was successful. I'm assuming a NULL return value?